integrations/server/test_covidcast_endpoints.py

```diff
@@ -53,20 +66,52 @@ def test_basic(self):
             self.assertEqual(out["result"], -1)
 
         with self.subTest("simple"):
-            out = self._fetch("/", signal=first.signal_pair(), geo=first.geo_pair(), time="day:*")
+            out = self._fetch("/", signal=first.signal_pair, geo=first.geo_pair, time="day:*")
+            self.assertEqual(len(out["epidata"]), len(rows))
+
+        with self.subTest("unknown signal"):
+            rows = [CovidcastRow(source="jhu-csse", signal="confirmed_unknown", time_value=20200401 + i, value=i) for i in range(10)]
+            first = rows[0]
+            self._insert_rows(rows)
```

Comment:
How is this different from subTest `simple`? In both cases, it looks like we're querying the data we just inserted. Did you mean to query for something that wasn't available in the db?

Reply:
Hm, good point. It doesn't look any different except it compares the values. I'll merge these two.